content/projects/ecft-vs-pipeline.md

```diff
@@ -0,0 +1,26 @@
+---
+title: ECFT-VS Pipeline
```

Comment:
I would put the hyphen between 'VS' and 'pipeline' rather than between 'ECFT-VS'. Like this, the 'VS' might read like part of the protein name.